bin/p2-dsctl/main.go

```diff
@@ -60,6 +61,14 @@ var (
 	updateMinHealth = cmdUpdate.Flag("minhealth", "The minimum health of the daemon set").String()
 	updateName      = cmdUpdate.Flag("name", "The cluster name (ie. staging, production)").String()
 	updatePodID     = cmdUpdate.Flag("pod", "The pod ID on the daemon set").String()
+
+	cmdOverrideSelector = kingpin.Command(CmdOverrideSelector, `
+		The raw selector for the daemon set represented as a string.
+		The selector string uses same syntax as the kubernetes selectors or p2-select without flags.
+		It is highly that recommended that you test it on p2-select first`,
```

Comment:
furthermore, `p2-select` is not an open source binary so we shouldn't reference it in open source code